### PR TITLE
Update index.html

### PR DESCRIPTION
**Pull request by Dan-Vanilla**
Hi Im dany 
I still don't know how to make tables, and I'm too tired to investigate more... I know I know U_U but I think tomorrow afternoon I can upload something more interesting. That s for now :)

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -2,38 +2,70 @@
...
 </head>
 <body>
-<center>
-<h1>Chef Brian Bevand</h1>
-<img src="chef.jpg" style="width: 50%; border-radius: 50%;" />
-<h1><a href="mailto:chef@brianbevand.com">chef@brianbevand.com</a></h1>
-<br />
-<table>
+<center><h1 class="title">Chef Brian Bevand</h1></center>
+<img src="chef.jpg" style="width: 35%; padding-left: 25px;" /> <article > <p class="extra-text"> hffhsdds sdvsv</p></article>
```
Comment (purpleidea) [inline index.html:57]: > hffhsdds sdvsv

?

File: index.html [hunk 2]
```diff
@@ -2,38 +2,70 @@
...
+}
+.extra-text {
+  display: inline-block;
+}
+footer {
+	font-size: 10px;
+	text-align: center;
+}
+.food {
+	width: 85%;
```
Comment (purpleidea) [inline index.html:30]: Is this indentation done with tabs or spaces?

While there may or may not be common use of one vs. the other in certain communities, when contributing to an existing project, it's usually best to follow whatever convention is already there unless there is a strong reason not to.

File: index.html [hunk 3]
```diff
@@ -2,38 +2,70 @@
...
-	border-collapse:collapse;
-	width: 100%;
-}
-td
-{
-	border: 0px;
-	border-collapse:collapse;
-	width: 25%;
+body{ 
+    background-image: url("veggies.jpg");
```
Comment (purpleidea) [inline index.html:6]: This photo isn't part of the patch.

File: index.html [hunk 4]
```diff
@@ -2,38 +2,70 @@
...
 	height: auto;
+	opacity: 100%;
+}
+.title {
+	font-family:impact;
+	color: white;
+	font-size: 60px;
+}
+.extra-text {
+  display: inline-block;
```
Comment (purpleidea) [inline index.html:23]: inconsistent indentation

File: index.html [hunk 5]
```diff
@@ -2,38 +2,70 @@
...
+
+a:hover {
+  color: skyblue;
+}
+
+a:active {
+  color: lightcyan;
 }
 </style>
+
```
Comment (purpleidea) [inline index.html:53]: What's this for?

**Review by purpleidea — CHANGES_REQUESTED** [review]
Overall, great to see you wrote your first patch! I've left some review comments.

For v2 of this patch (or close and create a new one) try and get all the tiny details 100% right. If I were to merge the patch, this would be live on the website, and it wouldn't make sense to do so as is.

Lastly, make sure your commit message describes what's going on with the patch, and that the patch has a real purpose! Keep at it.

File: index.html (hunk 1 above)
Comment (Dan-Vanilla) [reply index.html:57]: just a simulated text

File: index.html (hunk 2 above)
Comment (Dan-Vanilla) [reply index.html:30]: I created the "food" thing for my table's background

File: index.html (hunk 3 above)
Comment (Dan-Vanilla) [reply index.html:6]: ok

File: index.html (hunk 5 above)
Comment (Dan-Vanilla) [reply index.html:53]: I guess is not in the right place

File: index.html (hunk 4 above)
Comment (Dan-Vanilla) [reply index.html:23]: same. like food. 
So how do you customize parts of the site? what if I want a style for a text ??

Comment (Dan-Vanilla) [reply index.html:23]: I was putting classes for everything. I can put it whith properties right?

File: index.html (hunk 2 above)
Comment (purpleidea) [reply index.html:30]: Please re-read the previous comment.

File: index.html (hunk 4 above)
Comment (Dan-Vanilla) [reply index.html:23]: Oh I understand.

**Pull request merged**